src/main/java/com/bookstoreapp/controller/BookController.java

```diff
@@ -0,0 +1,36 @@
+package com.bookstoreapp.controller;
+
+import com.bookstoreapp.dto.BookDto;
+import com.bookstoreapp.dto.CreateBookRequestDto;
+import com.bookstoreapp.service.BookService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping(value = "api/books")
```

Comment:
extract `api/` to the application properties `server.servlet.context-path=/api` or somehow property in order not to duplicate it each time in each controller
```suggestion
@RequestMapping("/books")
```